### PR TITLE
UXG-1276 adds agents configure guided setup page

### PR DESCRIPTION
**Pull request by nicmarson**
http://jira.hq.idx.org/browse/UXG-1276

### CODE REVIEW

File: src/vue/backend/src/views/guided-setup/AgentsConfigure.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        guidedSetupMixin,
+        pageGuard
+    ],
+    components: {
+        AgentsSettings,
+        GuidedSetupContentCard
+    },
+    computed: {
+        ...mapState({
+            enabled: state => state.agentSettings.enabled,
```
Comment (sgreenl5) [inline src/vue/backend/src/views/guided-setup/AgentsConfigure.vue:35]: What are you using enabled for?

Comment (nicmarson) [reply src/vue/backend/src/views/guided-setup/AgentsConfigure.vue]: I knew I forgot something

**Review by ralls — APPROVED** [review]
Nice

**Review by sgreenl5 — CHANGES_REQUESTED** [review]
Clicking continue on `guided-setup/listings/advanced` should navigate to this view, but it does not

**Review by ralls — APPROVED** [review]

**Pull request merged**